lib/logstash/outputs/syslog.rb

```diff
@@ -111,6 +111,7 @@ def connect
   public
   def receive(event)
     return unless output?(event)
+    return if (event['tags'] || []).include?('_jsonparsefailure')
```

Comment:
In my opinion it is not the task of the output plugin to decide if an event should be handled or not. This should be done by conditionals (https://www.elastic.co/guide/en/logstash/current/event-dependent-configuration.html#conditionals).